src/components/features/Brand/Brand.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Brand.module.scss';
+import Button from '../../common/Button/Button';
+
+class Brand extends React.Component {
+  render() {
+    const { brands } = this.props;
+
+    return (
+      <div className={styles.root}>
+        <div className='container'>
+          <div className={`row ${styles.marksRow}`}>
+            <div className={styles.brandBox}>
+              <div className={`col ${styles.thumbnail}`}>
+                <Button className={styles.arrowButton}>
```

Comment:
Buttony powinny zajmować całą powierzchnię diva w który są owinięte, albo nie powinny się w nim w ogóle znajdować, w tym momencie przycisk to jedynie ikona strzałki w którą trzeba celować

Reply:
jasne, to do zrobienia przy wprowadzeniu funkcjonalności :)